Tests/StreamChatUITests/SnapshotVariation.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatUI
+import XCTest
+
+enum SnapshotVariation: String, Hashable, CaseIterable {
+    case small
+    case medium
+    case large
+    case extraLarge
+    case extraExtraExtraLarge
+    case light
+    case dark
+    
+    var trait: UITraitCollection {
+        switch self {
+        case .small: return UITraitCollection(preferredContentSizeCategory: .small)
+        case .medium: return UITraitCollection(preferredContentSizeCategory: .medium)
+        case .large: return UITraitCollection(preferredContentSizeCategory: .large)
+        case .extraLarge: return UITraitCollection(preferredContentSizeCategory: .extraLarge)
+        case .extraExtraExtraLarge: return UITraitCollection(preferredContentSizeCategory: .extraExtraExtraLarge)
+        case .light: return UITraitCollection(userInterfaceStyle: .light)
+        case .dark: return UITraitCollection(userInterfaceStyle: .dark)
+        }
+    }
+}
```

Comment:
This is great 👍 Let's make it even better:
- we can combine all those with a `displayScale = 1` trait and make the screenshots smaller -> faster comparison
- let's try to keep the number of screenshots to a minimum and combine those traits together. Maybe we just need [small + dark], [large + light],[xxl + dark]?
- any other interesting trait collecitons we should test for?

Reply:
1. Yes great idea 👍 That will also reduce the size of the screenshots 🚀 
2. Yeah we can for sure join some of them, to reduce the number of screenshots 👍 
3. For now I think we have the most important ones, and later when we have better accessibility features, we can use this [plugin](https://github.com/cashapp/AccessibilitySnapshot) of the snapshot framework to test more accessibility features. But maybe there's more that we can add right now? Thoughts?  @GetStream/ios-developers

Reply:
OK, let's start simple for now, we can always add more things as we need them.